public/components/event_analytics/redux/slices/query_slice.ts

```diff
@@ -28,7 +30,7 @@ const initialQueryState = {
   [PATTERN_REGEX]: PPL_DEFAULT_PATTERN_REGEX_FILETER,
   [FILTERED_PATTERN]: '',
   [SELECTED_TIMESTAMP]: '',
-  [SELECTED_DATE_RANGE]: ['now-40y', 'now'],
+  [SELECTED_DATE_RANGE]: [coreRefs.queryAssistEnabled ? QUERY_ASSISTANT_FIXED_START_TIME : NON_QUERY_ASSISTANT_START_TIME, 'now'],
```

Comment:
`coreRefs` are set in plugin start stage, could this const be initialized before that?

Reply:
i tried it and got `now-15m` with query assist enabled because coreRefs were initialized after query_slice import

Reply:
Changed back to the explicit now-15m timestamp, leaving the timestamp issues for another pr